Mathlib/Combinatorics/Quiver/ConnectedComponent.lean

```diff
@@ -50,18 +57,149 @@ instance [Inhabited V] : Inhabited (WeaklyConnectedComponent V) :=
   ⟨show V from default⟩
 
 protected theorem eq (a b : V) :
-    (a : WeaklyConnectedComponent V) = b ↔ Nonempty (@Path (Symmetrify V) _ a b) :=
+    (a : WeaklyConnectedComponent V) = b ↔
+      Nonempty (@Path (Symmetrify V) _ a b) :=
```

Comment:
Please undo the style changes here: what you have proposed is all fine but then: so is the original so it's easier if we avoid the churn.